src/components/CarouselCard/readme.md

```diff
@@ -48,3 +48,98 @@ const carouselContainerStyles = {
     </CarouselCard>
 </div>
 ```
+
+##### CarrouselCard with CarouselImage changed dynamically
+
+```js
+import React from 'react';
+import { 
+    CarouselCard,
+    CarouselImage,
+    ButtonIcon
+} from 'react-rainbow-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {faPlus} from '@fortawesome/free-solid-svg-icons';
+
+const carouselContainerStyles = {
+    maxWidth: 600,
+};
+
+class CarrouselCardExample extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isNewCardAdded: false,
+        };
+    }
+
+    addNewCard() {
+        const { isNewCardAdded } = this.state;
+        this.setState({ isNewCardAdded: !isNewCardAdded  });
+    }
+
+    renderNewCard() {
+        const { isNewCardAdded } = this.state;
+        if (isNewCardAdded) {
+            return (
+                <CarouselImage
+                    src="images/illustrations/Illustration-rainbow-6.svg"
+                    header="New Card"
+                    description="New card description."
+                    alternativeText="New card accessible description."
+                    href="/#/Components/CarouselImage"
+                />
+            );
+        }
+        return null;
+    }
+
+    render() {
+        return (
+            <div className="rainbow-p-bottom_xx-large">
+                <GlobalHeader className="rainbow-m-bottom_xx-large" src="images/user/user3.jpg">
+                    <ButtonIcon
+                        id="button-icon_add-new-card"
+                        className="rainbow-m-left_small"
+                        onClick={() => this.addNewCard()}
+                        variant="border"
+                        icon={<FontAwesomeIcon icon={faPlus} />}
+                    />
+                </GlobalHeader>
+                <CarouselCard
+                    style={carouselContainerStyles}
+                    className="rainbow-align-content_center rainbow-m_auto"
+                    id="carousel-3"
+                    disableAutoScroll={true}
```

Comment:
we usually only use:
```disableAutoScroll```
when passing true values as props
and when pass false:
```disableAutoScroll={false}```
here there is no problem because this is a markdown file that eslint doesn't inspect but in js files, the linter doesn't allow you to do this.